packages/redux/src/entities/schemas/productsList.ts

```diff
@@ -67,7 +67,7 @@ export default new schema.Entity(
           // If the filter segment is an ProductVariantAttribute, filteredFacetGroups will be an array with
           // all attributes so the flow is a little different.
           if (filterSegment.type === ATTRIBUTES_TYPE) {
-            let facetIndex;
+            let facetIndex: undefined | number;
```

Comment:
As the facetIndex is expected to be a `number` and not `undefined`, and across the project, we usually use `number | undefined`, can we change this to the suggestion below?

```suggestion
            let facetIndex: number | undefined;
```

Reply:
Done, thank you!